application/controllers/collection_instrument.py

```diff
@@ -85,20 +87,34 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
         :return: a collection instrument instance
         """
 
+        if current_app.config["SEFT_CI_DATABASE_TABLE_DEPRECATED"]:
+            try:
+                seft_ci_bucket = GoogleCloudSEFTCIBucket(current_app.config)
+                path = seft_ci_bucket.upload_file_to_bucket(file=file)
+            except Exception:
+                log.exception("An error occurred when trying to put SEFT CI in bucket")
+
         log.info("Upload exercise", exercise_id=exercise_id)
 
         validate_uuid(exercise_id)
         instrument = InstrumentModel(ci_type="SEFT")
 
-        seft_file = self._create_seft_file(instrument.instrument_id, file)
-        instrument.seft_file = seft_file
+        if current_app.config["SEFT_CI_DATABASE_TABLE_DEPRECATED"] is False:
```

Comment:
This won't work because it'll evaluate to 
```
if "false" is False
```
Which is False and not true

Reply:
You'll need to use the strtobool function to convert it to a boolean

https://github.com/ONSdigital/response-operations-ui/blob/57f78176c565f0d72acbfe2fcf69007831139f0b/config.py#L23

Reply:
Also this is quite hard to read as it's a double negative.

The feature flag might be better as SEFT_GCS_ENABLED. Then you could do
```
 if not current_app.config["SEFT_GCS_ENABLED"]:
```

which to me seems much easier to read